### PR TITLE
Add itemlore command

### LINKED ISSUE #1911: [Enhancement] Adding /itemlore to EssentialsX
### Information

Full output of `/ess version`:

Server log (upload `logs/latest.log` to [Gist](https://gist.github.com/)):

EssentialsX config (upload `plugins/Essentials/config.yml` to [Gist](https://gist.github.com/)): 

### Details

Describe your issue here.

I would like to suggest a new feature to EssentialsX which is renaming your Item and Itemlore. It's a good feature without adding separate plugins. Also an easier alternative for the vanilla rename.

### PR DESCRIPTION
**Pull request by JRoy**
Adds a command to add to or clear an items lore.

Closes #1911

### CODE REVIEW

**Comment by mdcfe** [comment]
It might be useful to allow users to modify or remove specific lines too, so they don't need to wipe the lore and start over.

**Review by pop4959 — CHANGES_REQUESTED** [review]
I tried out the command, and I like it, but setting a specific line's lore doesn't seem to work.

To replicate this, add several lines of lore, such as `/itemlore add a` three times. Then, try something like `/itemlore set 1 b`, and notice that the text doesn't change.

**Comment by JRoy** [comment]
mfw 
image

File: Essentials/src/com/earth2me/essentials/commands/Commanditemlore.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    public Commanditemlore() {
+        super("itemlore");
+    }
+
+    @Override
+    protected void run(Server server, User user, String commandLabel, String[] args) throws Exception {
+        ItemStack item = user.getBase().getItemInHand();
+        if (item.getType().name().contains("AIR")) {
+            user.sendMessage(tl("itemloreInvalidItem"));
+            return;
```
Comment (pop4959) [inline Essentials/src/com/earth2me/essentials/commands/Commanditemlore.java]:
```suggestion
            throw new Exception(tl("itemloreInvalidItem"));
```
This is probably better, to remain consistent with the way you are handling errors in other parts of this command.

**Review by pop4959 — CHANGES_REQUESTED** [review]

**Comment by JRoy** [comment]
@pop4959 changes applied

**Review by pop4959 — DISMISSED** [review]

**Review by pop4959 — DISMISSED** [review]

File: Essentials/src/messages.properties [hunk 2]
```diff
@@ -324,6 +324,14 @@ itemCannotBeSold=\u00a74That item cannot be sold to the server.
 itemCommandDescription=Spawn an item.
 itemCommandUsage=/<command> <item|numeric> [amount [itemmeta...]]
 itemId=\u00a76ID\:\u00a7c {0}
+itemloreClear=\u00a76You have cleared this item''s lore.
+itemloreCommandDescription=Edit the lore of an item.
+itemloreCommandUsage=/<command> <add/set/clear> [text/line] [text]
+itemloreInvalidItem=\u00a74You need to hold an item to edit its lore.
+itemloreNoLine=\u00a74Your selected item does not have lore text on line \u00a7c{0}\u00a74.
```
Comment (mdcfe) [inline Essentials/src/messages.properties]:
```suggestion
itemloreNoLine=\u00a74Your held item does not have lore text on line \u00a7c{0}\u00a74.
```

File: Essentials/src/plugin.yml [hunk 3]
```diff
@@ -216,6 +216,10 @@ commands:
     description: Searches for an item.
     usage: /<command> <item>
     aliases: [dura,edura,durability,edurability,eitemdb,itemno,eitemno]
+  itemlore:
+    description: Edit the lore of an item.
+    usage: /<command> <add/set/clear> [text/line] [text]
+    aliases: [ilore, eilore, eitemlore]
```
Comment (mdcfe) [inline Essentials/src/plugin.yml]:
```suggestion
    aliases: [lore, elore, ilore, eilore, eitemlore]
```

`/lore` seems like a reasonable alias - whereas `/name` for `/itemname` might be ambiguous, `/lore` seems to be more obviously related to items.

**Review by mdcfe — CHANGES_REQUESTED** [review]

**Review by pop4959 — APPROVED** [review]

**Review by mdcfe — APPROVED** [review]

**Pull request merged**